### PR TITLE
UI improvements - round 1

### PR DESCRIPTION
**Pull request by Jay-o-Way**
These should be "safe" changes.
More will come later, in different PRs, after more clarity

- general cleanup of XAML
- added **icons** on windows
- move **Button** to the bottom 
- Mica + ThemeWatcher
- use **Button** for browser controls
- accessibility (Screen reader)
- (temporarily?) moved **DropDown** button to the bottom because
  > This button cannot be used in this [top] position, in combination with a `wpfui:TitleBar`. The TitleBar will block the button.
  https://github.com/U-C-S/Hurl/discussions/131#discussioncomment-9961304

### Screenshots

image
image
image
image

### CODE REVIEW

**Comment by Jay-o-Way** [comment]
I realize this is still a bit large, but I just can't help it 🫣
Even after that other PR, I got some merge conflicts and that really annoys me.

**Comment by U-C-S** [comment]
😐 ... Well then, I'll take care of it then, after or before merging.

File: Source/Hurl.RulesetManager/Windows/EditRuleset.xaml [hunk 1]
```diff
@@ -115,64 +103,57 @@
                             <wpfui:TextBox
                                 x:Name="_Rule"
                                 Grid.Column="1"
-                                Margin="8" />
+                                HorizontalAlignment="Stretch" />
                         </Grid>
 
-                        <Grid Grid.Row="2">
+                        <Grid Grid.Row="2" Margin="0,0,0,8">
```
Comment (Jay-o-Way) [inline Source/Hurl.RulesetManager/Windows/EditRuleset.xaml:109]:
```suggestion
                        <Grid Grid.Row="2">
```

File: Source/Hurl.RulesetManager/Windows/EditRuleset.xaml [hunk 2]
```diff
@@ -183,27 +164,38 @@
...
+                        MinWidth="120"
+                        Margin="0,0,8,0"
+                        HorizontalAlignment="Center"
+                        Appearance="Primary"
+                        Click="SaveButton_Click"
+                        Content="Save" />
+                    <wpfui:Button
+                        MinWidth="120"
+                        HorizontalAlignment="Center"
+                        Appearance="Transparent"
```
Comment (Jay-o-Way) [inline Source/Hurl.RulesetManager/Windows/EditRuleset.xaml:194]:
```suggestion
```

File: Source/Hurl.RulesetManager/Windows/MainWindow.xaml [hunk 3]
```diff
@@ -11,56 +11,38 @@
...
-                        Click="TestButton_Click">
-                        Test
-                    </wpfui:Button>
-                </StackPanel>
+        <Border Grid.Row="2" Style="{DynamicResource BottomBarStyle}">
+            <StackPanel Orientation="Horizontal">
+                <wpfui:Button
+                    MinWidth="120"
+                    Margin="0,0,8,0"
+                    HorizontalAlignment="Stretch"
```
Comment (Jay-o-Way) [inline Source/Hurl.RulesetManager/Windows/MainWindow.xaml:37]:
```suggestion
```

File: Source/Hurl.RulesetManager/Windows/MainWindow.xaml [hunk 4]
```diff
@@ -11,56 +11,38 @@
...
+                    Click="CreateButton_Click"
+                    Content="Create" />
+                <wpfui:Button
+                    MinWidth="120"
+                    HorizontalAlignment="Stretch"
```
Comment (Jay-o-Way) [inline Source/Hurl.RulesetManager/Windows/MainWindow.xaml:42]:
```suggestion
```

**Review by Jay-o-Way — COMMENTED** [review]
Few details

**Pull request merged**